examples/messages/src/App.tsx

```diff
@@ -21,15 +22,15 @@ function App() {
         channel: "TEST",
         account: account,
         fileObject: file,
-        storageEngine: StorageEngine.STORAGE,
+        storageEngine: ItemType.storage,
```

Comment:
On aleph-sdk-ts version 1.0.6 (used in the demo app's `package.json`) this can't work as ItemType does not exist... (see previous comment)
```js
storageEngine: StorageEngine.STORAGE
```